joomla/joomla-cms
# [5.1] Convert mod_syndicate to service provider

### Summary of Changes

Converts the syndicate link site module to service provider.

### Testing Instructions

1. Make sure you have site module mod_syndicate published
2. Make sure you have RSS Feed link enabled in Articles component configuration (Integration tab)
3. You should see module output on articles listing menu items - Category blog, Featured articles...
4. You should not see module output on other menu item types - Single Article, other components...

![mod_syndicate](https://github.com/joomla/joomla-cms/assets/59166858/bbc810f4-b414-4820-a7c9-32643f25deb7)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
